### PR TITLE
Nethack 3.6.0 (update formula)

### PR DESCRIPTION
**Pull request by jthemphill**
The dev team really hit it out of the park with this one - the formula is much cleaner now.

### CODE REVIEW

File: nethack.rb [hunk 1]
```diff
@@ -1,73 +1,48 @@
 require "etc"
```
Comment (dunn) [inline nethack.rb]: I think we can remove this now.

**Comment by jthemphill** [comment]
I think there's something wrong with the CI. The Mavericks machine had an error pulling the changes but the other machines had no problems.

**Comment by dunn** [comment]
The Mavericks VM is really fussy.  I just re-queued it, let's see if that was a fluke.

**Comment by dunn** [comment]
Now it's having trouble merging the patch.  Can you try rebasing against master (and squashing while you're at it)? That might help.
```
==> Applying patch
Applying: nethack: update 3.6.0 bottle.
Using index info to reconstruct a base tree...
M	nethack.rb
Falling back to patching base and 3-way merge...
Auto-merging nethack.rb
CONFLICT (content): Merge conflict in nethack.rb
Failed to merge in the changes.
Patch failed at 0001 nethack: update 3.6.0 bottle.
The copy of the patch that failed is found in:
   /usr/local/Library/Taps/Homebrew/homebrew-games/.git/rebase-apply/patch
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".
Error: Patch failed to apply: aborted.

**Comment by jthemphill** [comment]
:rocket:

**Comment by stevestreza** [comment]
This formula works for me when installed on El Capitan (10.11.1).

**Comment by antifuchs** [comment]
Love this change! (I was working on a very similar one before I saw this open PR.)

There's one more thing that you can do in the formula that's probably worth doing: install the manual page.
``` ruby
(man+"man6").install "doc/nethack.6"
```
is what I did, and that makes `man nethack` work.

**Comment by fjarlq** [comment]
There's a build failure:

    Error: 1 problem in 1 formula
    homebrew/games/nethack:
     * "man+"man6"" should be "man6"

referring to:

https://github.com/jthemphill/homebrew-games/commit/5fdd33e803ad0abec49d6e8b41e9a05e7fd2ce66#diff-bb360424e07da8d57eec263df41b70a1R47

Looks like that line should read:

    man6.install "doc/nethack.6"

**Comment by mikaelengstrom** [comment]
:+1:

**Comment by mistydemeo** [comment]
Given the lack of action in #200, I'm going to go ahead and merge this after a rebuild.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**